tests/unit/dataactvalidator/test_a35_cross_file.py

```diff
@@ -0,0 +1,69 @@
+from tests.unit.dataactcore.factories.domain import SF133Factory
+from tests.unit.dataactcore.factories.staging import AppropriationFactory, ObjectClassProgramActivityFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+
+_FILE = 'a35_cross_file'
+_TAS = 'a35_cross_file_tas'
+
+
+def test_column_headers(database):
+    expected_subset = {'row_number', 'deobligations_recoveries_r_cpe', 'ussgl487100_downward_adjus_cpe_sum',
+                       'ussgl497100_downward_adjus_cpe_sum', 'ussgl487200_downward_adjus_cpe_sum',
+                       'ussgl497200_downward_adjus_cpe_sum'}
+    actual = set(query_columns(_FILE, database))
+    assert (actual & expected_subset) == expected_subset
+
+
+def test_success(database):
+    """ Tests that, for entries with the matching TAS, Appropriations deobligations_recoveries_r_cpe equals the sum of
```

Comment:
Thanks for this detailed docstring--really handy not to reference the rule text to understand what's happening here.